### ISSUE TITLE
Provide And and But as aliases for defineStep

### ISSUE THREAD

**Issue by JeremiePat** [cucumber/cucumber-js#1891]
**Is your feature request related to a problem? Please describe.**
The Gherkin syntax allow to use both `And` and `But` complementary steps

**Describe the solution you'd like**
It would be nice if Cucumber JS could expose `And` and `But` as aliases for `defineStep` just like `Given`, `When` and `Then`

**Describe alternatives you've considered**
I'm currently just doing:

```js
import {Given, When, Then, defineStep as And, defineStep as But} from '@cucumber/cucumber'
```
Which works but is quite ugly.

**Comment by jenisys** [comment]
You normally do not need "And" and/or "But" for your step implementations. These "keywords" are just used in Feature files (in Gherkin) to chain multiple steps of the same category: Either "Given" or "When" or Then".

Could you explain why you think that you need "And" and "But" ?

BACKGROUND:
Each step category has slightly different semantics:

* The "Given" steps set something up
* The "When" steps trigger an event or command
* The "Then" steps check/assert that a certain outcome is reached

With "And" and "But" you have a generic step which may lead to misunderstood semantics and wrong usage by others.

**Comment by davidjgoss** [comment]
This has been raised before, and it's still not something we're going to do. See https://github.com/cucumber/cucumber-js/issues/1118

TLDR: "And" and "But" are feature-file synonyms for "Then"; as such you should define steps as "Then"
